README.md

```diff
@@ -1,24 +1,75 @@
-# README
+# Members Only
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> An application that allows members to create posts. The authors of the posts are displayed only if a user is logged in.
 
-Things you may want to cover:
+## Functionality
 
-* Ruby version
+- User Account: Users can sign Up, sign in and sign out
+- Post: Users can create new posts and view a list of all posts.
 
-* System dependencies
+## :toolbox: Tools & Technologies Used
 
-* Configuration
+- Ruby v2.6.5
+- Ruby on Rails v5.2.4.3
+- SQLite3
 
-* Database creation
+## :rocket: Getting Started
 
-* Database initialization
+### Dependencies
 
-* How to run the test suite
+- Devise v4.7.2
 
-* Services (job queues, cache servers, search engines, etc.)
+### Configuration
 
-* Deployment instructions
+1. [Install Ruby](https://www.ruby-lang.org/en/documentation/installation/)
+2. Install Ruby on Rails
+```
+gem install rails -v 5.2.3
+```
+3. [Install SQLite3](https://www.sqlite.org/download.html), if you don't already have it installed locally.
+4. Clone repo into your local environment using your favourite terminal:
+```
+git clone [this-repo-url].git
+```
+5. Open project directory
+```
+cd [directory-name]
+```
+6. Bulk install dependencies
+```
+bundle
+```
+7. Set up database
+```
+rake db:migrate
```

Comment:
I think that before `rake db:migrate` there is generally `rails db:create`
Also, be more precise and indicate `bundle install`